### PR TITLE
Add new owner and reviewer for Vietnamese content

### PR DESCRIPTION
**Pull request by huynguyennovem**
Add **huynguyennovem** as an owner and reviewer for Vietnamese content

Signed-off-by: Nguyen Quang Huy <huynguyennovem@gmail.com>

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -182,9 +182,11 @@ aliases:
   sig-docs-vi-owners: # Admins for Vietnamese content
     - ngtuna
     - truongnh1992
+    - huynguyennovem
```
Comment (truongnh1992) [inline OWNERS_ALIASES]: You have to alphabetize the name of owners

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -182,9 +182,11 @@ aliases:
...
   sig-docs-vi-reviews: # PR reviews for Vietnamese content
     - ngtuna
     - truongnh1992
+    - huynguyennovem
```
Comment (truongnh1992) [inline OWNERS_ALIASES]: You have to alphabetize the name of owners

Comment (huynguyennovem) [reply OWNERS_ALIASES]: Thanks! I have just fixed this.

File: OWNERS_ALIASES (hunk 1 above)
Comment (huynguyennovem) [reply OWNERS_ALIASES]: Thanks! I have just fixed this.

**Comment by truongnh1992** [comment]
/approve 
@huynguyennovem has already become a member of Kubernetes org. Hope you will contribute more and more to K8s.

**Comment by huynguyennovem** [comment]
> /approve
> @huynguyennovem has already become a member of Kubernetes org. Hope you will contribute more and more to K8s.

Thank you @truongnh1992

**Comment by zacharysarah** [comment]
@truongnh1992 It looks like you may have manually merged this PR. Is that the case?

**Comment by truongnh1992** [comment]
@zacharysarah Yes, I merged this PR manually. @huynguyennovem is my co-worker at Vietnam Kubernetes Community, and he want to contribute to K8s document Vietnamese localization.

**Comment by zacharysarah** [comment]
@truongnh1992 Thanks for the clarification. Manual merges aren’t acceptable; write permissions are solely for creating long-running collaborative branches. 

Please confirm that you understand manual merges aren’t allowed, and will refrain from manually merging PRs in the future.

**Comment by truongnh1992** [comment]
> 
> 
> @truongnh1992 Thanks for the clarification. Manual merges aren’t acceptable; write permissions are solely for creating long-running collaborative branches.
> 
> Please confirm that you understand manual merges aren’t allowed, and will refrain from manually merging PRs in the future.

Thank you for your explain. I got it.
I'm sorry for this.

**Pull request merged**